### DISCUSSION TITLE
Welcome to codedocs.nvim Discussions!

### DISCUSSION THREAD

**Discussion by jeangiraldoo** [jeangiraldoo/codedocs.nvim#1] [Announcements]
## 👋 Hi there!
Discussions will be used to ask and answer questions about the project, make suggestions, or just connect with other people that contribute to or use codedocs.

**Comment by Methapon2001** [comment]
Is there a way to do something like jumping after codedocs insert? 

Like when I use codedocs on a function and it generate docs with parameters and I want to jump to each parameter and write description something like that. Kinda work like snippets?

I don't know much about Neovim API, I don't know if this is possible?

**Reply by jeangiraldoo** [reply]
Hi, I didn't know snippets were a thing if I'm honest with you, but it seems promising! Codedocs currently doesn't support that feature, but it's one of the planned features before reaching beta. I'll work on it after I get the hang of how snippets work (I'll install a plugin for that tomorrow).  I'll update this discussion when support for that is added
